spec/index.html

```diff
@@ -230,10 +230,10 @@ <h3>RDF Literals</h3>
         are used to identify values such as strings, numbers, dates.</p>
 
       <p>Literals (Grammar production <a href="#grammar-production-literal">Literal</a>)
-        have a lexical form followed by a
+        have a lexical form followed by either a
         <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
         (possibly including <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>),
-        a <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>,
+        or a <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>,
```

Comment:
This `or` is incorrect. `choose a, b, c, or d` not `choose a, or b, or c, or d`.
```suggestion
        a <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>,
```